docs/questions/query-builder/expressions-list.md

```diff
@@ -23,6 +23,8 @@ For an introduction to expressions, check out the [overview of custom expression
   - [Sum](#sum)
   - [SumIf](./expressions/sumif.md)
   - [Variance](#variance)
+  - [CumulativeSum](./expressions/cumulative.md)
+  - [CumulativeCount](./expressions/cumulative.md)
```

Comment:
Do we want these listed in both places? Here and in the window function section?

Reply:
Yes, we added the Window Functions section because Gleb requested it, but 1) if someone isn't a SQL user, they don't know what a "window function" is, and 2) even I myself, a person who knows what window functions are and that these pages are under that section, was still looking for them under "Aggregations" because cumulative metrics don't really "feel" like window functions (the window is the entire dataset)